CorsixTH/Lua/game_ui.lua

```diff
@@ -554,11 +554,19 @@ function GameUI:onMouseMove(x, y, dx, dy)
 
   if self:_isMouseScrollButtonDown() then
     local zoom = self.zoom_factor
-    self.current_momentum.x = -dx/zoom
-    self.current_momentum.y = -dy/zoom
+    self.current_momentum.x = self.current_momentum.x - dx/zoom
+    self.current_momentum.y = self.current_momentum.y - dy/zoom
+
+    local momentum_x_int = math.floor(self.current_momentum.x + 0.5)
```

Comment:
We locally support `math.round` if you would prefer that.

Reply:
Ok, yes, i think `math.round` is clearer, I have updated the code to use that, thanks